Packs/CommonScripts/Scripts/GetInstances/GetInstances.py

```diff
@@ -0,0 +1,43 @@
+from typing import Callable, Dict, Iterator
+
+import demistomock as demisto
+from CommonServerPython import *
+
+INTERNAL_MODULES_BRANDS = ['Scripts', 'Builtin', 'testmodule']
+ENABLED = 'active'
+DISABLED = 'disabled'
+
+
+def prepare_filter_list(args: Dict) -> List[Callable[[Dict], bool]]:
```

Comment:
this solution is nice but it is more complex than it should, please simplify it.